### ISSUE TITLE
An error occurred (2). Please try again.

### ISSUE THREAD

**Issue by Ojigrande** [infinyte7/Manjaro-Linux-on-Android#10]
Hello.

I have tried to use your script to install Manjaro on my tablet, but I get the error:
```An error occurred (2). Please try again.```

**Comment by deniscerri** [comment]
same bug on my device too
Galaxy S10 Snapdragon

**Comment by infinyte7** [comment]
Without physical devices it is hard to debug the error.
I have tested it locally on all Android devices and it working.
I may find more about error after testing more devices.

**Comment by infinyte7** [comment]
@deniscerri @Ojigrande 
Use termux wakelock and keep screen lock 
Because in background download stop so it is giving error.

**Comment by mrfoxie** [comment]
> @deniscerri @Ojigrande Use termux wakelock and keep screen lock Because in background download stop so it is giving error.

still it's give the same error 
![Screenshot_20220307-222820_Termux](https://user-images.githubusercontent.com/25217451/157081604-13c77639-8bbb-4bb0-ba81-f14852be9e04.png)

**Comment by krmanik** [comment]
Run the following in termux. It will solve the error.
```
pkg update 
pkg upgrade
```

**Comment by mrfoxie** [comment]
> Run the following in termux. It will solve the error.
> 
> ```
> pkg update 
> pkg upgrade
> ```

noo use I have already done that also I reinstall the app and script but still shows that error

**Comment by mrfoxie** [comment]
I think we should close this issue because I have resolved the problem by upgrading all the packages.

**Comment by shim80** [comment]
What do you meant?
I also upgraded everything using pkg upgrade && pkg update, but that doesn't change anything. 
I have a Galaxy Note 9

**Comment by shim80** [comment]
I did everything, still doesn't work.

**Comment by mrfoxie** [comment]
> I did everything, still doesn't work. 

It should work on all the device, I don't know why it's not working on your device. Just paste the following code and paste that log here
```
termux-info > log.md
````

**Comment by deniscerri** [comment]
What fixed it for me, was removing termux completely and updating the packages before trying to install it. Also make sure you are using the fdroid one, not the play store version

**Comment by mrfoxie** [comment]
> I did everything, still doesn't work.

give me some time I will make a video on it so that you can see that steps for installing manjaro

**Comment by mrfoxie** [comment]
@krmanik  I think this may be a bug because when I'm using my cell network then its works perfectly but as soon as I shift my network to any Wi-Fi connection then it give me error. So this is my theory that may be @Ojigrande or I'm using Wi-Fi then its not allowing me to download it. @Ojigrande can you tell me what kind of network your using while installing because I'm using my own cellular network to install Manjaro on my termux this will be really helpful to identify the problem.

**Comment by krmanik** [comment]
I think it may be/may be not the reason. Because I have just installed it over Wi-Fi.

**Comment by mrfoxie** [comment]
@krmanik I agree with your point but I also got the same error while I'm trying to install it on the Wi-Fi network after following all the steps I have mentioned above. I tried changing 2-3 Wi-Fi and got the same error so its is the problem of network in my case.

**Comment by AndreasBrostrom** [comment]
Getting this error as well newly installed termux from F-droid.

Done above attempts with pkg and no effect either.

**Comment by AndreasBrostrom** [comment]
solved it, manually installing proot, pv and pulseaudio

`pkg install -y proot pv pulseaudio`

allow you to continue to install.

**Comment by ItsAleph** [comment]
I had same issue, and it resolved absolutely randomly and without any reason (i just re-ran manjaro.sh a few times). woo

**Comment by ssaagamemer** [comment]
just tried pkg install -y proot pv pulseaudio ?
it work perfectly fine on me when using this command

**Comment by ReflexTheLegend** [comment]
same issue here...

**Comment by zolex** [comment]
Same issue. None of the solutions provider here help
